### PR TITLE
gh-14 Add support for Einstein GA release-train

### LINKED ISSUE #14: Add support for Einstein GA release-train
The train includes the app listed [here](https://github.com/spring-cloud-stream-app-starters/app-starters-release/releases/tag/vEinstein.RELEASE).

~Depends on #12.~

### PR DESCRIPTION
**Pull request by ghillert**
resolves https://github.com/spring-cloud-stream-app-starters/initializr/issues/14

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -120,28 +136,28 @@ initializr:
...
           groupId: org.springframework.cloud.stream.app
           artifactId: field-value-counter-app-dependencies
           additionalBoms: [app-starters-core-bom]
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
-            version: 2.0.0.RELEASE
-            repositories: spring-snapshots,spring-milestones
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```
Comment (sobychacko) [inline src/main/resources/application.yml:148]: We dropped field-value-counter in the 2.1.0.RELEASE.

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -70,8 +78,8 @@ initializr:
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
-            version: 2.0.0.RELEASE
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```
Comment (sobychacko) [inline src/main/resources/application.yml:81]: We dropped aggregate-counter in the 2.1.0.RELEASE.

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -120,28 +136,28 @@ initializr:
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
-            version: 2.0.0.RELEASE
-            repositories: spring-snapshots,spring-milestones
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```
Comment (sobychacko) [inline src/main/resources/application.yml:139]: We dropped counter in the 2.1.0.RELEASE.

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -397,7 +480,7 @@ initializr:
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```
Comment (sobychacko) [inline src/main/resources/application.yml:483]: No tasklauncher-cloudfoundry in 2.1.0.RELEASE.

File: src/main/resources/application.yml [hunk 5]
```diff
@@ -407,7 +490,7 @@ initializr:
...
```
Comment (sobychacko) [inline src/main/resources/application.yml:493]: No tasklauncher-local apps in 2.1.0.RELEASE.

File: src/main/resources/application.yml [hunk 6]
```diff
@@ -417,7 +500,7 @@ initializr:
...
```
Comment (sobychacko) [inline src/main/resources/application.yml:503]: No tasklauncher-yarn apps in 2.1.0.RELEASE.

File: src/main/resources/application.yml (hunk 4 above)
Comment (sobychacko) [inline src/main/resources/application.yml:483]: This is not in 2.1.0.RELEASE.

File: src/main/resources/application.yml (hunk 1 above)
Comment (ghillert) [reply src/main/resources/application.yml:148]: `- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`

File: src/main/resources/application.yml (hunk 2 above)
Comment (ghillert) [reply src/main/resources/application.yml:81]: `- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`

File: src/main/resources/application.yml (hunk 3 above)
Comment (ghillert) [reply src/main/resources/application.yml:139]: `- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`

File: src/main/resources/application.yml (hunk 4 above)
Comment (ghillert) [reply src/main/resources/application.yml:483]: `- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`

File: src/main/resources/application.yml (hunk 5 above)
Comment (ghillert) [reply src/main/resources/application.yml:493]: `- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`

File: src/main/resources/application.yml (hunk 6 above)
Comment (ghillert) [reply src/main/resources/application.yml:503]: `- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`

File: src/main/resources/application.yml (hunk 4 above)
Comment (ghillert) [reply src/main/resources/application.yml:483]: `- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`

**Review by sobychacko — APPROVED** [review]

**Pull request merged**